src/classes/info.py

```diff
@@ -47,30 +50,38 @@
 EXPORT_PRESETS_PATH = os.path.join(PATH, "presets")
 
 # User paths
-HOME_PATH = os.path.join(os.path.expanduser("~"))
-USER_PATH = os.path.join(HOME_PATH, ".openshot_qt")
-BACKUP_PATH = os.path.join(USER_PATH)
-RECOVERY_PATH = os.path.join(USER_PATH, "recovery")
-THUMBNAIL_PATH = os.path.join(USER_PATH, "thumbnail")
-CACHE_PATH = os.path.join(USER_PATH, "cache")
-BLENDER_PATH = os.path.join(USER_PATH, "blender")
-TITLE_PATH = os.path.join(USER_PATH, "title")
-TRANSITIONS_PATH = os.path.join(USER_PATH, "transitions")
-EMOJIS_PATH = os.path.join(USER_PATH, "emojis")
-PREVIEW_CACHE_PATH = os.path.join(USER_PATH, "preview-cache")
-USER_PROFILES_PATH = os.path.join(USER_PATH, "profiles")
-USER_PRESETS_PATH = os.path.join(USER_PATH, "presets")
-USER_TITLES_PATH = os.path.join(USER_PATH, "title_templates")
-PROTOBUF_DATA_PATH = os.path.join(USER_PATH, "protobuf_data")
-YOLO_PATH = os.path.join(USER_PATH, "yolo")
+OLD_HOME_PATH = os.path.join(os.path.expanduser("~"))
+OLD_USER_PATH = os.path.join(OLD_HOME_PATH, ".openshot_qt")
+
+HOME_PATH = QStandardPaths.writableLocation(QStandardPaths.HomeLocation)
+USER_PATH = QStandardPaths.writableLocation(QStandardPaths.DataLocation)
+APP_PATH  = os.path.join(USER_PATH, NAME)
```

Comment:
At the risk of bikeshedding, rather than `APP_PATH` I'd prefer to call this something like `CONFIG_PATH` or `SETTINGS_PATH`. "`APP_PATH`" sounds too much like the _install_ location — it doesn't clearly indicate "**user** data".

Reply:
Also, _currently_ — for reasons I'm not entirely clear on — the application name is being set to `openshot` (not `openshot-qt` _or_ `OpenShot Video Editor`) in `launch.py`. Which means things like `QStandardPaths.AppDataLocation` and `QStandardPaths.AppConfigLocation` will be `<DataLocation>/openshot/`, and `<ConfigLocation>/openshot/` once the QApplication instance is set up. To avoid possibly shooting ourselves in the foot for later uses of QStandardPaths, we should probably use compatible pathnames. (Whether that means changing `launch.py` or `classes.info`, I could go either way.)

Reply:
<s>In fact... currently, the only reasons `classes.info` is loaded before the application instance is set up are:

1. To test loading of the application modules, or adjust `sys.path` if it fails. The `sys.path` manipulation is terrible and we need to get rid of it _anyway_, but in the mean time we can certainly use something other than `classes.info` as our module-loading canary.
1. To get the program version from `info.SETUP['version']`. But that's the wrong way to handle that anyway. The version should be stored in the `__version__` attribute of the package. We can move that to `__init__.py` where it belongs.
1. To hold some configuration stuff used by the command line args. But I can move that stuff somewhere else, like a `classes.config` module. It doesn't need to be in `classes.info`.
1. To display `info.PATH` on the console output. That's silly and can be moved.
1. It'll also get loaded when `classes.sentry` is imported, but only so _it_ can load `info.VERSION` and `info.CURRENT_LANGUAGE`. Again, those can be migrated elsewhere and we can remove that dependency.

If we fix those few exposures, we can hold off loading `classes.info` until after we've set the application name, and then we could use paths like `QStandardPaths.AppConfigDir`, `.AppCacheDir`, `.App[Local]DataDir` directly in `classes.info`, instead of having to ad-hoc them. I kind of like that possibility, myself.</s>

...Actually, never mind. I just tried it, and it's perfectly OK to call these _before_ the application instance is created:
```python3
from PyQt5.QtCore import QCoreApplication
QCoreApplication.setApplicationName('something')
QCoreApplication.setApplicationVersion('1.2.3-four5')
```

...So, we can (and probably should) just do that at the top of `classes.info`, instead of where it's located now in `launch.py`, and then use the `App`-specific versions of the `QStandardPaths`. (Or,  we could move the app name setting to the very top of `launch.py`, before it imports `classes.info`. Same effect, and it's already hardcoded so it's not like it needs `classes.info`.)

Reply:
(Also confirmed...)
```python3
from PyQt5.QtCore import QCoreApplication, QStandardPaths
QCoreApplication.setApplicationName('openshot')
print(QCoreApplication.writableLocation(QStandardPaths.AppConfigLocation))
# ==> '/home/ferd/.config/openshot/`
```